src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -509,17 +509,31 @@ def _job_run (self, jd) :
 
         # target host specifica
         # FIXME: these should be moved into resource config files
+        self._logger.debug ("submit SLURM script to %s", self.rm)
         if 'bridges' in self.rm.host.lower():
 
             if total_gpu_count: 
+                # gres resouurces are specified *per node*
+                assert(self._ppn), 'need unique number of cores per node'
+                number_of_nodes = int(math.ceil(float(total_cpu_count) / self._ppn))
+                count = total_gpu_count / number_of_nodes
                 if cpu_arch: gpu_arch = cpu_arch.lower()
                 else       : gpu_arch = 'p100'
-                slurm_script += "#SBATCH --gres=gpu:%s:%s\n" % (gpu_arch, total_gpu_count)
+                slurm_script += "#SBATCH --gres=gpu:%s:%s\n" % (gpu_arch, count)
```

Comment:
Thanks, @iparask , good catch - the same actually holds for tiger...

Reply:
The `count` is the number of requested GPUs per node, so isn't this what we want? It might be more or less than 2, where more than 2 will lead to an error, which it should.  Am I missing something?